Makefile

```diff
@@ -32,11 +32,12 @@ GDB_PROFILE_INTERVAL ?= 0.1
 AUTO_TEST ?= none
 EXTRA_BLOCKLISTS_DIRS ?= ""
 SYSCALL_TEST_DIR ?= /tmp
+FEATURES ?= none
```

Comment:
This is misleading. Does `FEATURES = none` mean no features at all? It does not. It means the _default_ features are enabled. Cargo features are additive. So specifying this `FEATURES` variable means having some extra features enabled. And it just happens that `all` is a feature that contains all other features. We should allow the user to specify features other than `all`. For example, setting `FEATURES=cvm_guest` should only turn on the `cvm_guest` feature.

Consider the following revised version.

```Makefile
FEATURES ?=

ifdef FEATURES
CARGO_OSDK_ARGS += --features="$(FEATURES)"
endif
```